extensions/amp-story/1.0/test/test-amp-story-quiz.js

```diff
@@ -277,5 +299,28 @@ describes.realWin(
         'i-amphtml-story-quiz-option-selected'
       );
     });
+
+    it('should preprocess percentages preserving ties, order, and adding to 100 (in most cases)', () => {
```

Comment:
Could you
- Add a more end to end test (that doesn't call any private method)
- Additional calls to `preprocessPercentages_` are fine if you want to test more cases to do tie breaking etc, but can you split them into different test cases? It's good to "only assert one thing per test". If this test were to fail, we wouldn't really know which part we broke

Reply:
Done!